pytorch_lightning/__about__.py

```diff
@@ -1,7 +1,7 @@
 import time
 
 _this_year = time.strftime("%Y")
-__version__ = '1.3.7'
+__version__ = '1.3.7post'
```

Comment:
I think it needs this:
```suggestion
__version__ = '1.3.7post0'
```